i18n/extra/es.json

```diff
@@ -93,8 +93,8 @@
             "SMW_YM"
         ],
         [
-            "SMW_MDY",
             "SMW_DMY",
+            "SMW_MDY",
```

Comment:
@mwjames thanks for caring, but, I'm from Brazil :brazil:: I speak Portuguese, not Spanish. :wink: 
Just for record, brazilians use the date format DMY as well.